config/dev.webpack.config.js

```diff
@@ -7,10 +7,15 @@ const insightsProxy = {
   ...(process.env.BETA && { deployment: 'beta/apps' }),
 };
 
+const ciOrStage = process.env.CI ? 'ci' : 'stage';
+const betaOrStable = process.env.BETA ? 'beta' : 'stable';
+// for accessing prod-beta change this to 'prod-beta'
+const env = `${ciOrStage}-${betaOrStable}`;
```

Comment:
It's a bit more complicated, there's also `prod` and `qa` environment.

```JS
const environment = proccess.env.CI ? 'ci' : (process.env.ENVIRONMENT || 'stage')
const betaOrStable = process.env.BETA ? 'beta' : 'stable';

const env = `${ciOrStage}-${betaOrStable}`;
```

This will allow running:
```bash
ENVIRONMENT=prod npm run start:proxy
```